projects/grpc-gateway/build.sh

```diff
@@ -15,5 +15,13 @@
 #
 ################################################################################
 
+mkdir $GOPATH/src/github.com/grpc-ecosystem
+mv $SRC/grpc-gateway $GOPATH/src/github.com/grpc-ecosystem/
+cd $GOPATH/src/github.com/grpc-ecosystem/grpc-gateway && go get ./...
 
-compile_go_fuzzer github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/httprule Fuzz fuzz
+if [ "$SANITIZER" = "coverage" ]
+then
+	compile_go_fuzzer github.com/grpc-ecosystem/grpc-gateway/internal/httprule Fuzz fuzz gofuzz
```

Comment:
It looks pretty suspicious to me that we are compiling some other code for coverage fuzzers.
Can you explain why we do this?